packages/pluggableWidgets/accordion-web/tests/e2e/specs/Accordion.spec.ts

```diff
@@ -0,0 +1,64 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+
+describe("accordion-web", () => {
+    beforeAll(() => {
+        page.open(); // resets page
+    });
+    it("compares with a screenshot baseline and checks if all accordion elements are rendered as expected", () => {
+        browser.setWindowRect(0, 0, 1200, 900);
+        const screenshotElem = $(".mx-name-container2");
+        screenshotElem.waitForDisplayed({ timeout: 5000 });
+        browser.saveElement(screenshotElem, "accordionPageContent");
+
+        expect(browser.checkElement(screenshotElem, "accordionPageContent")).toEqual(0);
+    });
+    it("hides group when the visibility is false", () => {
+        const accordionGroup = $(".mx-name-accordion1 > section:nth-child(3)");
+        const button = $(".mx-name-actionButton1");
+
+        expect(accordionGroup.getText()).toBe("Secondary");
+
+        button.click();
+
+        expect(accordionGroup.getText()).toBe("Success (custom header)");
+    });
+    it("shows first group content", () => {
+        const accordionGroup = $(".mx-name-accordion1 > section");
+        const accordionGroupContent = accordionGroup.$(".mx-name-text5");
+
+        expect(accordionGroupContent.isDisplayed()).toBeFalse();
+
+        accordionGroup.click();
+
+        expect(accordionGroupContent.isDisplayed()).toBeTrue();
+    });
+    it("shows one image in the group content", () => {
+        const accordionGroup = $(".mx-name-accordion1 > section:nth-child(2)");
+        const accordionGroupContent = accordionGroup.$(".mx-name-image1");
+
+        expect(accordionGroupContent.isDisplayed()).toBeFalse();
+
+        accordionGroup.click();
+
+        expect(accordionGroupContent.isDisplayed()).toBeTrue();
+    });
+    it("shows multiple accordions expanded", () => {
```

Comment:
Can we add another spec for singleExpanded mode, where only one group can be expanded at a time? It would probably be very similar to this test, but maybe with 3 groups instead of 1. 
1. Then it would start off with something like: 
```
expect(firstAccordionGroupContent.isDisplayed()).toBeFalse();
expect(secondAccordionGroupContent.isDisplayed()).toBeFalse();
expect(thirdAccordionGroupContent.isDisplayed()).toBeFalse();
```
2. Then you do
```
firstAccordionGroup.click();
```
3. Verify that:
```
expect(firstAccordionGroupContent.isDisplayed()).toBeTrue();
expect(secondAccordionGroupContent.isDisplayed()).toBeFalse();
expect(thirdAccordionGroupContent.isDisplayed()).toBeFalse();
```
4. Click on another one that was not the previous:
```
thirdAccordionGroup.click();
```
5. And then verify only that one is opened and the other ones are closed:
```
expect(firstAccordionGroupContent.isDisplayed()).toBeFalse();
expect(secondAccordionGroupContent.isDisplayed()).toBeFalse();
expect(thirdAccordionGroupContent.isDisplayed()).toBeTrue();
```

Reply:
Thanks! I followed your suggestion but I'm using a different approach because I had some timing issues using isDisplayed.  😁

Reply:
Looks good, nice work 😄

Reply:
Discussed with @leonardomendix over Slack, but he's gonna try to see if we can use the actual content as selectors rather than CSS classes, because that will lead to less flaky tests.

Reply:
Done!  :)